### PR TITLE
refactor(chart): Remove unnacessary env from engine & experiment CR

### PR DESCRIPTION
**Pull request by ispeakc0de**
Signed-off-by: shubhamchaudhary <shubham.chaudhary@mayadata.io>

-  Remove unnacessary env from engine & experiment CR

### CODE REVIEW

**Review by ksatchit — DISMISSED** [review]

**Comment by ksatchit** [comment]
@rajdas98 - pls check for coredns impact

**Review by uditgaurav — DISMISSED** [review]

**Review by ksatchit — DISMISSED** [review]

File: charts/coredns/coredns-pod-delete/engine.yaml [hunk 1]
```diff
@@ -29,7 +29,4 @@ spec:
         
             # set chaos interval (in sec) as desired
             - name: CHAOS_INTERVAL
-              value: '10'
-                
-            - name: CHAOS_NAMESPACE
-              value: 'kube-system'
```
Comment (imrajdas) [inline charts/coredns/coredns-pod-delete/engine.yaml]: I think we need this env and kube-system namespace is the default and mandatory namespace

**Review by imrajdas — CHANGES_REQUESTED** [review]

File: charts/coredns/coredns-pod-delete/engine.yaml (hunk 1 above)
Comment (ispeakc0de) [reply charts/coredns/coredns-pod-delete/engine.yaml]: Is this namespace is different from the namespace where we will install our chaos-engine for this experiment??

Comment (imrajdas) [reply charts/coredns/coredns-pod-delete/engine.yaml]: so, if we don't pass will it take chaos engine namespace?

Comment (ispeakc0de) [reply charts/coredns/coredns-pod-delete/engine.yaml]: yes, we are passing engine_ns from runner directly

**Review by imrajdas — DISMISSED** [review]

**Review by ksatchit — APPROVED** [review]

**Pull request merged**